sdk/tables/Azure.Data.Tables/api/Azure.Data.Tables.netstandard2.0.cs

```diff
@@ -121,6 +121,10 @@ public partial class TableServiceClient
         public virtual Azure.Response SetProperties(Azure.Data.Tables.Models.TableServiceProperties properties, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response> SetPropertiesAsync(Azure.Data.Tables.Models.TableServiceProperties properties, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
+    public static partial class TablesFilter
```

Comment:
When do we call it Table vs Tables?

Reply:
In this case I went with the plural because `TableFilter` might indicate that it is only for tables, rather than table entities. What do you think?

Reply:
Looking through the entire API file, it feels kind of weird to me because only other things with `Tables` are the namespace and the `ModelFactory`.  (And the `ModelFactory` might be incorrectly named - both `ModelFactory` and `ClientOptions` should have the same prefix.)

But after glancing through everything, would this make more sense as `public static string CreateQueryFilter(System.FormattableString filter) { throw null; }` next to all the other methods of the same name?  It might make it clearer that they're doing the same job and you can choose whichever is easier for you to write your query.